scico/linop/radon_svmbir.py

```diff
@@ -11,41 +11,52 @@
 `svmbir <https://github.com/cabouman/svmbir>`_ package.
 """
 
+from typing import Optional
+
 import numpy as np
 
 import jax
 import jax.experimental.host_callback
 
+import scico.numpy as snp
+from scico.loss import WeightedSquaredL2Loss
+from scico.typing import JaxArray, Shape
+
+from ._linop import LinearOperator
+
 try:
     import svmbir
 except ImportError:
     raise ImportError(
         "Could not import svmbir, please refer to INSTALL.rst "
-        "for instructions on how to install the SVMBIR."
+        "for instructions on how to install it."
     )
 
-import scico.numpy as snp
-from scico.loss import WeightedSquaredL2Loss
-from scico.typing import JaxArray, Shape
-
-from ._linop import LinearOperator
-
 
 class ParallelBeamProjector(LinearOperator):
-    r"""Parallel beam projector based on svmbir."""
+    r"""Parallel beam projector based on svmbir.
+
+    Perform tomographic projections of an image at specified angles.
+    """
 
     def __init__(
         self,
         input_shape: Shape,
         angles: np.ndarray,
         num_channels: int,
+        is_masked: Optional[bool] = False,
     ):
         """
         Args:
             input_shape: Shape of the input array.
             angles: Array of projection angles in radians, should be
-              increasing.
+                increasing.
             num_channels: Number of pixels in the sinogram
+            is_masked:  If True, the valid region of the image is
```

Comment:
Not obvious to me what "valid region" means. Is something set to zero? When?

Reply:
How about:

is_masked:  If True, the valid region of the image is
                determined by a mask defined as the circle inscribed
                within the image boundary.
                Otherwise, the whole image array is valid
                for projection and reconstruction.
                Pixels outside the valid region is not projected or updated during the reconstruction.

Reply:
Updated doc-string to make it clearer